src/translations/screens/subscreens/PrivacySettingsTexts.ts

```diff
@@ -1,59 +1,71 @@
-import { translation as _ } from '../../commons';
+import {translation as _} from '../../commons';
```

Comment:
Are there any changes in this file, or just changes by prettier?

Reply:
Just prettier